circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/DiffGeneratedPartitionPredicateTest.java

```diff
@@ -106,17 +105,23 @@ public void setUp() throws Exception {
   public void autogeneratePredicate() throws Exception {
     when(replica.getTableAndStatistics(tableReplication)).thenReturn(replicaTableAndStats);
     when(replicaTableAndStats.getTable()).thenReturn(table2);
+    when(sourceTable.getPartitionLimit()).thenReturn((short) 10);
+
+    predicate = new DiffGeneratedPartitionPredicate(source, replica, tableReplication, checksumFunction);
 
     assertThat(predicate.getPartitionPredicate(),
-        is("(p1='value1' AND p2='value2') OR (p1='value11' AND p2='value22')"));
+        is("(p1='value11' AND p2='value22') OR (p1='value1' AND p2='value2')"));
```

Comment:
I can't make up my mind whether this would be considered a breaking change. Technically I think it is, but I guess most of our end users wouldn't notice?

Reply:
I was doubting if we should make it configurable, but to be honest i you're using hiveDiff and your whole table is copied its all the same. It's just for the backfill, I think it's not a breaking change.

Reply:
OK, then let's just document it in the CHANGELOG. If there is an easy place to mention the order that HiveDiff processes the partitions in the README that would be nice but not required. We could add a feature to make it configurable in the future if anyone ever actually needs it.